packages/experimental/Icon/src/Icon.tsx

```diff
@@ -41,6 +41,7 @@ const fontStyleMemoCache = getMemoCache();
 
 function renderFontIcon(iconProps: IconProps) {
   const fontSource: FontIconProps = iconProps.fontSource;
+  const accessible = iconProps.accessible === undefined ? true : iconProps.accessible;
```

Comment:
nit: this could be determined using `iconProps.accessible ?? true`

The [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) returns the right-hand side when the left-hand side is null or undefined.

Reply:
We should also be changing the accessible prop on line 23 right (for raster images)?

Reply:
> nit: this could be determined using `iconProps.accessible ?? true`
> 
> The [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) returns the right-hand side when the left-hand side is null or undefined.

Yes, but in case of false the condition still will return true. This would work well if the default value would be false

Reply:
> We should also be changing the accessible prop on line 23 right (for raster images)?

yeah :) we obviously need to update Icon component. I'm looking forward to finish Avatar and to start Icon

Reply:
> > nit: this could be determined using `iconProps.accessible ?? true`
> > The [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) returns the right-hand side when the left-hand side is null or undefined.
> 
> Yes, but in case of false the condition still will return true. This would work well if the default value would be false

No, it'll return false. The ?? operator respects "false" as a value for the left hand side. You can play around with it on the page that Krystal linked. It will only use the right hand side if the left hand value if specifically undefined or null.

Reply:
> > We should also be changing the accessible prop on line 23 right (for raster images)?
> 
> yeah :) we obviously need to update Icon component. I'm looking forward to finish Avatar and to start Icon

Also we should modify the icon before we ship Avatar so that Avatar has the correct accessibility states. All of our components need to be accessible in the expected manner before shipping.

Reply:
> > > nit: this could be determined using `iconProps.accessible ?? true`
> > > The [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) returns the right-hand side when the left-hand side is null or undefined.
> > 
> > 
> > Yes, but in case of false the condition still will return true. This would work well if the default value would be false
> 
> No, it'll return false. The ?? operator respects "false" as a value for the left hand side. You can play around with it on the page that Krystal linked. It will only use the right hand side if the left hand value if specifically undefined or null.

Yeah, I tried